contracts/access/mod.go

```diff
@@ -0,0 +1,172 @@
+// Package access implements a native contract to handle access. It allows an
```

Comment:
Should we keep adding a `// Documentation Last Review: 01.02.2021` as @Gilthoniel had suggested?
It might be useful :)

... same comment in other files too :)